generate_plugin_placeholder.rb

```diff
@@ -0,0 +1,58 @@
+require "clamp"
+require "erb"
+require "octokit"
+require_relative "git_helper"
+
+class GeneratePluginPlaceholder < Clamp::Command
+  option "--output-path", "OUTPUT", "Path to a directory where logstash-docs repository will be cloned and written to", required: true
+  option "--plugin-type", "STRING", "Type (ex: integration, input, etc) of a new plugin.", required: true
+  option "--plugin-name", "STRING", "Name for a new plugin.", required: true
+
+  SUPPORTED_TYPES = %w(integration)
+
+  def execute
+    generate_placeholder(plugin_type, plugin_name)
+    submit_pr
+  end
+
+  # adds an empty static page under the VPR
+  # this helps us to eliminate broken link issues in the docs
+  def generate_placeholder(type, name)
+    if type.nil? || name.nil?
+      $stderr.puts("Plugin type and name are required.")
+      return
+    end
+
+    unless SUPPORTED_TYPES.include?(type)
+      $stderr.puts("#{type} is not supported. Supported types are #{SUPPORTED_TYPES.inspect}")
+      return
+    end
+
+    placeholder_asciidoc = "#{logstash_docs_path}/docs/versioned-plugins/#{type}s/#{name}-index.asciidoc"
```

Comment:
Let's add a check for the existence of this `logstash_docs_path`. IF the user doesn't pass a folder that contains a `logstash-docs` folder, then the error is not very helpful at the moment:

```
❯ ruby generate_plugin_placeholder.rb --output-path=//tmp --plugin-type=integration --plugin-name=testing
Errno::ENOENT: No such file or directory - //tmp/logstash-docs/docs/versioned-plugins/integrations/testing-index.asciidoc
               sysopen at org/jruby/RubyIO.java:1227
```

Reply:
Thank you, updated.